calculators/referendum_supporters_calculator.rb

```diff
@@ -1,6 +1,8 @@
 class ReferendumSupportersCalculator
-  def initialize(candidates: [], ballot_measures: [])
+  def initialize(candidates: [], ballot_measures: [], committees: [])
     @ballot_measures = ballot_measures
+    @committees_by_filer_id = committees
+      committees.where('"Filer_ID" IS NOT NULL').index_by { |c| c.Filer_ID }
```

Comment:
This looks wrong to me, I don't think the WHERE will be applied to the original query.

You should remove the `committees` on line 5